### ISSUE TITLE
Class attributes are not accessible when using GenericActor

### ISSUE THREAD

**Issue by rakanalh** [Bogdanp/dramatiq#15]
If you define:
```
class DoSomething(GenericActor):
    MODE_ACTION_1 = 0
    MODE_ACTION_2 = 1
```
And then try to use one of these attributes, an error will be raised:
```
AttributeError: 'Actor' object has no attribute 'ACTION_1'
```

**Comment by Bogdanp** [comment]
Thanks! This is fixed in 0.14.

**Comment by rakanalh** [comment]
Cool, thanks!
